pkg/report/fuzz.go

```diff
@@ -34,7 +34,7 @@ func Fuzz(data []byte) int {
 		}
 		switch os {
 		case "openbsd", "netbsd", "fuchsia":
-			// openbsd/netbsd has Start/End/SkipPos set incorrectly due to messing with /r/n.
+			// openbsd/netbsd has Start/End/SkipPos set incorrectly due to messing with \r\n.
```

Comment:
openbsd/netbsd can now be removed from the switch. Not sure about fuchsia.
The reason we skipped part of the checks is that Start/EndPos were set for the output without \r and did not match real output with \r.

Reply:
fixed.